Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -281,4 +281,15 @@
     <!-- XTXT: Signature validation dialog FAQ lint-->
     <string name="dcc_signature_validation_dialog_faq_link">https://www.coronawarn.app/de/faq/#hc_signature_invalid</string>
 
+    <!-- Expiration notification -->
+    <!-- XTXT: expiring soon notification -->
+    <string name="dcc_notification_expires_soon_text">Überprüfen Sie die Gültigkeit Ihrer Zertifikate.</string>
+    <!-- XTXT: expired notification -->
+    <string name="dcc_notification_expired_text">Überprüfen Sie die Gültigkeit Ihrer Zertifikate.</string>
+
+    <!-- DCC notification channel-->
+    <!-- XTXT: certificate notification channel title -->
+    <string name="dcc_notification_channel_title">Zertifikate</string>
+    <!-- XTXT: certificate notification channel description -->
+    <string name="dcc_notification_channel_description">Benachrichtigungen für Zertifikate verwalten</string>
```

Comment:
Does this need a `.` at the end?

Reply:
/cc @NataliaLemmerth ?

Reply:
@d4rken Since it's not a complete sentence, I think we do not need a period at the end.